MySolution.rb

```diff
@@ -0,0 +1,48 @@
+class MySolution
+
+    #problem1
+    def sumOfEven(array)
+        sum = 0
```

Comment:
So c#/java like :) can you come up with a 1 liner?

Steps:
- how can you drop `sum = 0` and `return sum`? do you even need a `return` statement in ruby?
- what are some alternatives to `for`?
- condense code.

Reply:
Well I admit that C# is may favorite language :) and Ruby is truly new to me and I just got started learning it - And Yeah I can try to make it in one line.

- For dropping `sum = 0` and the `return sum` I can use an array function from ruby `array.sum` and to sum over only the even values something like `array.select{|v| v.even?}` (that looks like a LINQ from C#)
- If you referring to loop alternatives to `for` in ruby I know about `while`, `until`,`begin-while`

Reply:
Well I admit that C# is may favorite language :) and Ruby is truly new to me and I just got started learning it - And Yeah I can try to make it in one line.

-For dropping `sum = 0` and the return sum I can use an array function from ruby `array.sum` and to sum over only the even values something like `array.select{|v| v.even?}` (that looks like a LINQ from C#)
-If you referring to loop alternatives to for in ruby I know about `while`, `until`,`begin-while`